tests/e2e/k8s.go

```diff
@@ -51,6 +51,17 @@ func applyK8sResourceManifest(kubectlOptions k8s.KubectlOptions, manifestPath st
 	k8s.KubectlApply(GinkgoT(), &kubectlOptions, manifestPath)
 }
 
+// checkExistenceOfK8sResource queries a Resource by it's kind, namespace and name and
+// returns the output of stderr
+func checkExistenceOfK8sResource(
+	kubectlOptions k8s.KubectlOptions,
+	resourceKind string,
+	resourceName string,
+) error {
```

Comment:
**Recommended**: This **SHOULD** be a bool function `isExistingK8sResource()` that is expected to be true wherever we want to assert that.
Such a function would be usable to assert not existing stuff as well.
I thought we already have a similar function in this file, but haven't checked.

Reply:
What you mean is getK8SResources I'd guess, we've had hat discussian before with Márton, we decided against using that but I'm not sure why, I don't really care either way, I'll use that.

Reply:
I think it is good to have this simplified function to check a resource existance. But we need the bool version

Reply:
Yeah, I lso tried getK8SResource and it would be a lot uglier to use that, so I'll keep using this.